app/helpers/spotlight/rendering_helper.rb

```diff
@@ -3,7 +3,11 @@
 module Spotlight
   module RenderingHelper # :nodoc:
     def render_markdown(text)
-      GitHub::Markup.render('.md', text).html_safe
+      # Extend Redcarpet renderer to add HTML anchors to each heading in the output HTML
+      renderer = Redcarpet::Render::HTML.new(with_toc_data: true)
+      # Use Redcarpet to render markdown as html
+      html = Redcarpet::Markdown.new(renderer).render(text)
+      sanitize(html, attributes: %w[href id])
```

Comment:
Is there a reason to use `sanitize` here instead of marking the response from `render` as `html_safe`?